flybywiresim/a32nx
# chore: more range in default calibration

## Summary of Changes

Adapted default calibration values (once again) to reduce support requests from users without calibrated controls. Also ahould avoid the need for controller and keyboard users to perform a calibration at all.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions


 - Delete the ThrottleConfiguration.ini file from the "work" folder in "Packages\Microsoft.FlightSimulator_8wekyb3d8bbwe\LocalState\packages\flybywire-aircraft-a320-neo\work". The path may vary for Steam installations.

 - Use as many as possible different devices (Keyboard, Controller, Throttle Quadrants) and check that all detents are snapping in correctly. 
 - Verify that "LVR CLB" is not flashing after takeoff, after pulling the throttle back into the CLB detent.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
